src/runtime/pkg/katautils/config.go

```diff
@@ -822,7 +822,10 @@ func newClhHypervisorConfig(h hypervisor) (vc.HypervisorConfig, error) {
 		return vc.HypervisorConfig{}, err
 	}
 
-	sharedFS := config.VirtioFS
+	sharedFS, err := h.sharedFS()
```

Comment:
Here are some virtiofs/nydusd checks for qemu:  
https://github.com/kata-containers/kata-containers/blob/cf53ec2c714d3dd665806c07d92120ae869f1573/src/runtime/pkg/katautils/config.go#L647-L655 
CLH also needs them.

Reply:
have updated